.run-tests.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 set -e
-module=''
-if [ ! -z "$@" ]; then
-    module=".$@"
+modules='tests'
+if [ ! -z "$*" ]; then
+    modules="$*"
 fi
 
 export PYTHONPATH="src"
-green --run-coverage tests"$module"
+green --run-coverage $modules
 
 # only report coverage if we're running a complete set of tests
-if [ -z "$module" ]; then
+if [ "tests integrations_tests" = "$modules" ]; then
```

Comment:
not sure if this line is behaving as expected, but it's not a deal-breaking to merging.